client/landing/stepper/declarative-flow/internals/steps-repository/__user/index.tsx

```diff
@@ -82,6 +85,7 @@ const UserStepComponent: StepType = function UserStep( {
 
 	const stepContent = (
 		<>
+			{ !! queryArgs.get( 'oneTapAuth' ) && ! notice && <OneTapAuthLoader /> }
```

Comment:
I wonder if we can move this within the `SignupFormSocialFirst` component because we probably want this outside of Stepper too. What I'm trying to say is that it's not step-specific.

Reply:
Sounds reasonable 👍🏻